packages/patternfly-4/react-core/src/components/ClipboardCopy/ClipboardCopy.d.ts

```diff
@@ -1,16 +1,18 @@
-import { FunctionComponent, HTMLProps, MouseEvent, ReactNode, HTMLDivElement } from 'react';
-import { OneOf } from '../../typeUtils';
+import { FunctionComponent, HTMLProps, MouseEvent, ReactNode } from 'react';
+import { OneOf } from '../../helpers/typeUtils';
 import { PopoverPosition } from '../Popover';
 
-export const ClipboardCopyVariant = {
-  inline: 'inline',
-  expansion: 'expansion'
-};
+export enum ClipboardCopyVariant {
```

Comment:
Why are we making this an enum?  We probably want to be consistent with other components that use a const.

Reply:
The issue cited this reason: "A 'const' initializer in an ambient context must be a string or numeric literal or literal enum reference."

Reply:
Can you verify that we are seeing this error in our typescript app?  I'm not sure we should make this change.

Reply:
There is a enum here https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/src/components/Title/Title.tsx#L14

Reply:
Trying to figure out exactly why it's throwing this error.  We use constants in other places, so wondering if you will see this error again.

Reply:
you can also set this to:
```
export const ClipboardCopyVariant : {
  inline = 'inline',
  expansion = 'expansion'
}
```
to resolve this

Reply:
the `=` throws it off for some reason